src/devices/machine/mb89374.cpp

```diff
@@ -0,0 +1,576 @@
+// license:BSD-3-Clause
+// copyright-holders:Ariane Fugmann
+/**
+	MB89374
+
+	Fujitsu
+	Data Link Controller
+
+ **/
+
+#include "emu.h"
+#include "emuopts.h"
+#include "mb89374.h"
```

Comment:
Please include the module header immediately after emu.h to help avoid nasty cross-dependency issues.